src/inim.nim

```diff
@@ -474,6 +492,8 @@ proc main(nim = "nim", srcFile = "", showHeader = true,
     indentSpaces = ""
     sessionNoAutoIndent = noAutoIndent
 
+  app.editor = getEnv("EDITOR")
```

Comment:
This doesn't work for me:

```
nim> getEnv "EDITOR"
 == type TaintedString
nim> 

Quitting INim: Goodbye!
 λ echo $EDITOR
nvim
```

Why is nim not picking up `$EDITOR` on my machine I wonder...

Reply:
```
./inim
👑 INim 0.5.0
Nim Compiler Version 1.2.0 [Linux: amd64] at /home/oatman/.nimble/bin/nim
nim> 
No $EDITOR set in ENV
```

Reply:
Hmmm, what do you get from `echo $EDITOR; ./inim`? And does `EDITOR=vim ./inim` correctly set EDITOR to `vim`?

Reply:
Oh wow, my fish shell is broken sorry! This works great in zsh :laughing: